runtime/pkg/duckdbsql/duckdbsql.go

```diff
@@ -48,50 +178,41 @@ type Annotation struct {
 	Value string
 }
 
-// ExtractAnnotations extracts annotations from comments prefixed with '@', and optionally a value after a ':'.
-// Examples: "-- @materialize" and "-- @materialize: true".
-func ExtractAnnotations() ([]*Annotation, error) {
-	panic("not implemented")
-}
-
 // ColumnRef has information about a column in the select list of a DuckDB SQL statement
 type ColumnRef struct {
-	Name      string
-	Expr      string
-	IsAggr    bool
-	IsStar    bool
-	IsExclude bool
-}
-
-// ExtractColumnRefs extracts column references from the outermost SELECT of a DuckDB SQL statement
-func ExtractColumnRefs(sql string) ([]*ColumnRef, error) {
-	panic("not implemented")
+	Name         string
+	RelationName string
+	Expr         string
+	IsAggr       bool
+	IsStar       bool
+	IsExclude    bool
 }
 
 // queryString runs a DuckDB query and returns the result as a scalar string
-func queryString(qry string, args ...any) (string, error) {
+func queryString(qry string, args ...any) ([]byte, error) {
 	rows, err := query(qry, args...)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
-	var res string
+	var res []byte
 	if rows.Next() {
 		err := rows.Scan(&res)
 		if err != nil {
-			return "", err
+			return nil, err
 		}
 	}
 
 	err = rows.Close()
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
 	return res, nil
 }
 
 // Use a global in-memory DuckDB connection for invoking DuckDB's json_serialize_sql and json_deserialize_sql
+// TODO: Why not get driver connection?
```

Comment:
Since AST parsing doesn't depend on any DB state, just seemed nicer to decouple things